### PR TITLE
Download Excel Data

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

Allows users to download the facilities list as an `.xlsx` file, in addition to `.csv`.

Connects #955 

## Demo

Screen Shot 2021-03-30 at 10 17 39 AM

## Testing Instructions

* `vagrant ssh` and run `./scripts/update`
* Run `./scripts/server`
* Navigate to [:6543](http://localhost:6543/) and click 'download'
     - [x] You should see a dropdown with CSV and Excel
* Click Excel
     - [x] An XLSX file of the data should be downloaded

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

**Comment by jwalgran** [comment]
Looking at this now.

File: src/app/src/components/DownloadFacilitiesButton.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import { logDownload } from '../actions/logDownload';
+
+const downloadFacilitiesStyles = Object.freeze({
+    listHeaderButtonStyles: Object.freeze({
+        height: '45px',
+        margin: '5px 0',
+    }),
+});
+
+function NonVectorTileFilterSidebarFacilitiesTab({
```
Comment (jwalgran) [inline src/app/src/components/DownloadFacilitiesButton.jsx:19]: For consistency we should rename this to `DownloadFacilitiesButton` to match this to the name of the file.

File: src/app/src/util/util.js [hunk 2]
```diff
@@ -68,6 +75,18 @@ export const downloadListItemCSV = (list, items) =>
 export const downloadFacilitiesCSV = (facilities, options) =>
     DownloadCSV(createFacilitiesCSV(facilities, options), 'facilities.csv');
 
+export const createFacilitiesXLSX = (facilities, options) => {
+    const data = formatDataForCSV(facilities, options);
+    const worksheet = XLSX.utils.aoa_to_sheet(data);
+    const workbook = XLSX.utils.book_new();
+    XLSX.utils.book_append_sheet(workbook, worksheet, 'Facilities');
```
Comment (jwalgran) [inline src/app/src/util/util.js:82]: Consider the lowercase `facilities` as the sheet name, which is consistent with the name of the file `facilities.csv` and the all lowercase column headers.

**Review by jwalgran — APPROVED** [review]
Works perfectly. I verified CSV download still works and then downloaded facilities from Turkey as an XLSX, which often have unicode characters in names and addresses. I was able to open and view the XLSX in LibreOffice, Numbers, and Google Sheets and see the unicode characters correctly.

**Comment by TaiWilkin** [comment]
Thanks for the review!

**Pull request merged**